cupy/core/core.pyx

```diff
@@ -1858,16 +1858,19 @@ cdef bint _is_hip = runtime._is_hip_environment
 cdef int _cuda_runtime_version = -1
 cdef str _cuda_path = ''  # '' for uninitialized, None for non-existing
 
-cdef list _cupy_header_list = [
+cdef list cupy_header_list = [
```

Comment:
This is a `cdef` variable defined in `.pyx`. I don't think we need to mark it private (by prepending an underscore), as it already is. No functions (either in Cython or Python) outside this module can access it.